haystack/components/eval/statistical_evaluator.py

```diff
@@ -30,68 +28,38 @@ class Metric(Enum):
         F1 = "F1"
```

Comment:
c.f https://github.com/deepset-ai/haystack/pull/6982#discussion_r1489857959 and https://github.com/deepset-ai/haystack/pull/6982#discussion_r1489859090